package.json

```diff
@@ -93,7 +93,7 @@
     "lint:ts": "yarn lint packages/*/src",
     "lint:versions": "node scripts/verifyPatternflyVersions.js",
     "lint:tests": "yarn lint packages/*/src/components/*/__tests__/*.test.*",
-    "prepare": "ts-patch install -s",
+    "postinstall": "ts-patch install -s && husky",
```

Comment:
The new version of Yarn actually does not support `prepare`, so I had to use the `postinstall` hook instead.